src/SeleniumLibrary/keywords/cookie.py

```diff
@@ -55,24 +55,55 @@ def get_cookie_value(self, name):
 
     @keyword
     def get_cookie(self, name):
-        """Returns the cookie found with `name` in extended variable format.
-
-        If no cookie is found with `name`, this keyword fails.
+        """Returns a cookie object found with ``name``.
+
+        If no cookie is found with ``name``, keyword fails. The cookie object
+        contains details about the cookie. Attributes available in the object
+        are documented in the table below.
+        | = Attribute = |             = Explanation =                            |
+        | domain        | Specifies hosts to which the cookie will be sent       |
+        | expiry        | The maximum lifetime of the cookie as EPOCH            |
+        | httpOnly      | HttpOnly cookie cannot be accessed by client-side APIs |
```

Comment:
This is a boolean, right? Types should be mentioned at least when they aren't strings.

Reply:
Yes it's a boolean. Good point about the documentation.

Reply:
Done